metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLModelProperties.pdl

```diff
@@ -93,24 +117,24 @@ record MLModelProperties includes CustomProperties, ExternalReference {
   deployments: optional array[Urn]
 
   /**
-   * List of jobs (if any) used to train the model
+   * List of jobs or process instances (if any) used to train the model
    */
   @Relationship = {
     "/*": {
       "name": "TrainedBy",
-      "entityTypes": [ "dataJob" ],
+      "entityTypes": [ "dataJob", "dataProcessInstance" ],
       "isLineage": true
     }
   }
   trainingJobs: optional array[Urn]
 
   /**
-   * List of jobs (if any) that use the model
+   * List of jobs or process instance (if any) that use the model
    */
   @Relationship = {
     "/*": {
       "name": "UsedBy",
-      "entityTypes": [ "dataJob" ],
+      "entityTypes": [ "dataJob" , "dataProcessInstance" ],
```

Comment:
This is where we can remove, to ensure that Data Process Instance Outputs contains this.